src/main/java/Quiz/QuizService.java

```diff
@@ -0,0 +1,125 @@
+package Quiz;
+
+import Quiz.dto.QuizDto;
+import Quiz.dto.QuizIdDto;
+import Quiz.model.Quiz;
+import User.DBUser;
+import Util.HibernateUtil;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("quiz")
+public class QuizService {
+
+    @GET
+    @Path("{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getQuiz(@PathParam("id") int id){
+        Session session = HibernateUtil.getSession();
+        Quiz quiz = session.get(Quiz.class, id);
+
+        if (quiz == null)
+            return Response.status(Response.Status.NOT_FOUND).build();
+
+        QuizIdDto dto = new ObjectMapper().convertValue(quiz, new TypeReference<QuizIdDto>(){});
+        session.close();
+        return Response.status(Response.Status.OK).entity(dto).build();
+    }
+
```

Comment:
Can we create QuizDAO and extract all hibernate interaction from the service to this? In this way the DAO is concerned with Hibernate and the service is concerned with REST :) 
Same applies to User